db/seeds/baseline_rates.seeds.rb

```diff
@@ -0,0 +1,28 @@
+require 'activerecord-import/base'
+require 'activerecord-import/active_record/adapters/postgresql_adapter'
+
+require_relative 'support/domestic_400ng'
```

Comment:
[This `support/domestic_400ng.rb` file](#diff-4df36d327ccc7450a65c8a9a73e9df78) contains all the code from the various files previously in `db/seeds/baseline_rates/`.